controllers/admin/roomManagement.controller.js

```diff
@@ -323,12 +326,19 @@ export const manualCheckout = async (req, res) => {
       user: req.user
```

Comment:
is `checkoutTime` on line 319 in IST or UTC? we're comparing against 11:00:00, but that's assuming that `moment()` will return time in IST.